app/components/container/containerController.js

```diff
@@ -117,22 +117,28 @@ function ($scope, $state, $stateParams, $filter, Container, ContainerCommit, Ima
   };
 
   $scope.confirmRemove = function () {
+    var title = 'You are about to remove a container.';
     if ($scope.container.State.Running) {
-      ModalService.confirmDeletion(
-        'You are about to remove a running container.',
-        function (confirmed) {
-          if(!confirmed) { return; }
-          $scope.remove();
-        }
-      );
-    } else {
-      $scope.remove();
+      title = 'You are about to remove a running container.';
     }
+    var text = 'Automatically clean associated volumes<i></i>';
+    ModalService.confirmContainerDeletion(
+      title,
+      text,
+      function (result) {
+        if(!result) { return; }
+        var cleanAssociatedVolumes = false;
+        if (result[0]) {
```

Comment:
result is undefined if user click cancel
Result is an empty array [] if user click remove without checkbox
result is ["1"] if user click remove with checkbox